config/settings.py

```diff
@@ -547,8 +547,8 @@
 
 # Debug Toolbar
 # -----------------------------------------------------------------------------
-DEBUG_TOOLBAR_PATCH_SETTINGS = False
 INTERNAL_IPS = ["127.0.0.1", "::1"]
+DEBUG_TOOLBAR_CONFIG = {"SHOW_COLLAPSED": True}
```

Comment:
The debug panel always loads collapsed for me on my dev instance, but for the a11y tests it was always expanded. Not sure why the difference, but the panel has a lot of a11y failures when expanded (increased the total count by ~25%...)

Reply:
I'm guessing is that it uses cookie or some other storage to remember last state (collapsed/expanded).